eo-maven-plugin/src/test/java/org/eolang/maven/ResolveMojoTest.java

```diff
@@ -66,4 +70,86 @@ public void testSimpleResolve(@TempDir final Path temp) throws Exception {
         );
     }
 
+    @Test
+    public void testConflictingDependencies(@TempDir final Path temp) {
+        final Exception excpt = Assertions.assertThrows(
+            IllegalStateException.class,
+            () -> simulateConflicts(temp, false)
+        );
+        MatcherAssert.assertThat(
+            excpt.getMessage(),
+            Matchers.equalTo(
+                String.format(
+                    "%s%s",
+                    "Conflicting dependencies are found for org.eolang:eo-runtime:jar",
+                    ":: [0.22.0, 0.22.1]"
+                )
+            )
+        );
+    }
+
+    @Test
+    public void testConflictingDependenciesNoFail(@TempDir final Path temp) throws IOException {
+        simulateConflicts(temp, true);
+        MatcherAssert.assertThat(
```

Comment:
@mximp the necessity to write this dummy code is the indicator that something is wrong in the design of the test. the procedure (function with no result) is the bad design: `simulateConflicts`

Reply:
Removed the method.